spec/models/iiif_record_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe IIIFRecord, :type => :model do
+  let(:solr_doc) {{
+    "title_citation_display" => ['This is the Title'],
+    "summary_note_display"   => ['This is a note about it.'],
+    "pub_date_display"       => ['1970'],
+    "language_facet"         => ['English', 'Spanish'],
+    "language_code_s"        => ['eng'],
+    "author_roles_1display"  => ['{"secondary_authors":[],"translators":["Translator, Bob"],"editors":[],"compilers":[],"primary_author":"Author, Alice"}']
+  }}
+  subject { described_class.new solr_doc }
+  it 'produces iiif json' do
+    iiif_json = {
+      label: 'This is the Title',
+      description: 'This is a note about it.',
+      metadata: [
+        { label: 'date', value: '1970' },
+        { label: 'language', value: ['English', 'Spanish'] },
```

Comment:
We've never been that clear about this in the spec because there are so many permutations. There was a [thread on IIIF discuss recently](https://groups.google.com/d/msg/iiif-discuss/OAa1uhaIJ6s/H7aCdgDkAAAJ).

I don't see why you'd need all those extra nodes if you're not going provide any meta-metadata.

```
{
  label: 'language',
  value: [
    {
      '@value': 'English'
    },
    {
       '@value': 'Spanish'
    }
  ]
}
```